R/sparsevctrs.R

```diff
@@ -1,3 +1,22 @@
+#' Using sparse data with recipes
+#' 
+#' [recipe()], [prep()], and [bake()] all accept sparse tibbles from the 
+#' `sparsevctrs` package and sparse matrices from the `Matrix` package. Sparse
+#' matrices are converted to sparse tibbles internally as each step expects a 
+#' tibble as its input, and is expected to return a tibble as well.
+#' 
+#' Several steps work with sparse data. A step can either work with  sparse 
+#' data, ruin sparsity, or create sparsity. The documentation for each  step 
+#' will indicate whether it will work with sparse data or create sparse columns. 
+#' If nothing is listed it is assumed to ruin sparsity.
+#' 
+#' Spare tibbles or data.frames will be return from [bake()] if sparse columns 
+#' are present in data, either from being generated in steps or because sparse 
+#' data was passed into [recipe()], [prep()], or [bake()].
```

Comment:
```suggestion
#' Sparse tibbles or `data.frame`s will be returned from [bake()] if sparse columns 
#' are present in data, either from being generated in steps or because sparse 
#' data was passed into [recipe()], [prep()], or [bake()].
```

I think this is saying that sometimes, when supplied sparse data at bake, steps will materialize and others they won't? As in, `data.frame`s don't support sparsity? As written, it's unclear whether "Sparse" modifies "tibbles" or "tibbles or data.frames." Maybe 

> Supplied a sparse tibble, [bake()] will either return a `data.frame` or sparse tibble...

This ^^^ also assumes that, in "if sparse columns are present in data," we're not referring to sparse matrices.